assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -47,10 +47,20 @@ const BoxedNumeric = styled(Numeric)`
   border-radius: 4px;
 `;
 
+const StyledToggle = styled(Toggle)`
+  + span {
+    margin: 0 10px;
+  }
+`;
+
 function isNum(v) {
   return typeof v === 'number' && !isNaN(v);
 }
 
+function formatRotationAngleNumber(v) {
+  return v > 360 ? v % 360 : v;
```

Comment:
Maybe better limit the input to 359 for example? This means when the user input 38 and then 0, it will changed to 20. Maybe it confusing users?